public/app/plugins/datasource/prometheus/querybuilder/parsing.test.ts

```diff
@@ -53,8 +53,17 @@ describe('buildVisualQueryFromString', () => {
       buildVisualQueryFromString(
         'avg(rate(access_evaluation_duration_count{instance="host.docker.internal:3000"}[$__rate_interval]))'
       )
-    ).toEqual(
-      noErrors({
+    ).toEqual({
+      // @todo, determine if this error node is expected after migration from lezer-promql to @prometheus-io/lezer-promql and upgrading @lezer/common to 1.0.0
+      errors: [
```

Comment:
@aocenas Do you know why the changes in this PR are resulting in an error node, but not currently on main (when parsing the same string?)

Reply:
It seems that the updated parser is throwing an error node when parsing the `__interval` `__rate_interval` and  variables, but everything still appears to work correctly with those variables on this branch (as far as I can determine with my limited knowledge).

If we can verify that functionality related to these variables are working as expected on this branch, then could we argue the difference in test results are expected behavior of the (updated) parsing library, and these changes to the tests are correct?

Reply:
Ok, as far as I can tell $__rate_interval is still working properly, and since the parsed trees are still returning the same output (just with additional error nodes), so we're going to assume that the updated parser is more verbose in reporting errors. 

If you are from the future and searching for `prometheus $__rate_interval not working` I'm sorry